docs/concepts/islands.md

```diff
@@ -14,15 +14,14 @@ island. The file must have a default export that is a regular Preact component.
 ```tsx
 // islands/MyIsland.tsx
 
-import { useState } from "preact/hooks";
+import type { Signal } from "@preact/signals";
```

Comment:
Is this a typo? Should be `useSignal` instead?

Reply:
Yep, this is a typo. I'll submit a fix. Thanks for spotting!